lms/djangoapps/verify_student/management/commands/date_range_resubmit.py

```diff
@@ -0,0 +1,123 @@
+        "Retries SoftwareSecurePhotoVerifications passed as "
+        "arguments, or if no arguments are supplied, all that "
+        "are in a state of 'submitted' from a start_datetime to an end_datetime"
+    )
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--start_datetime',
+            action='store',
+            dest='start_datetime',
+            help='Start date for a date range of SoftwareSecurePhotoVerifications; '
+                 'Should be formatted as YYYY-mm-dd HH:MM:SS; '
+                 'Requires the --end_datetime arg.',
+        )
+
+        parser.add_argument(
+            '--end_datetime',
+            action='store',
+            dest='end_datetime',
+            help='End date for a date range of SoftwareSecurePhotoVerifications; '
+                 'Should be formatted as YYYY-mm-dd HH:MM:SS; '
+                 'Requires the --start_datetime arg.',
+        )
+
+        parser.add_argument(
+            '--batch_size',
+            action='store',
+            dest='batch_size',
+            type=int,
+            default=300,
+            help='Maximum number of SoftwareSecurePhotoVerifications to process. '
+                 'This helps avoid overloading the database while updating large amount of data.'
+        )
+
+        parser.add_argument(
+            '--sleep_time',
+            action='store',
+            dest='sleep_time',
+            type=int,
+            default=10,
+            help='Sleep time in seconds between update of batches'
+        )
+
+    def handle(self, *args, **options):
+        # Make sure we have both a start date and end date
+        try:
+            start_datetime = datetime.datetime.strptime(options['start_datetime'], '%Y-%m-%d %H:%M:%S')
+        except:
+            log.exception("--start-date argument not present")
+            return
+
+        try:
+            end_datetime = datetime.datetime.strptime(options['end_datetime'], '%Y-%m-%d %H:%M:%S')
+        except:
+            log.exception("--end-date argument not present")
+            return
+
+        log.info(
+            f"\nDEBUG:"
+            f"\nstart_datetime = {start_datetime}"
+            f"\nend_datetime = {end_datetime}"
+            f"\n\n"
+        )
+
+        batch_size = options['batch_size']
+        sleep_time = options['sleep_time']
+
+
+        # Get those 'submitted' objects in that date range
+        attempts_to_retry = SoftwareSecurePhotoVerification.objects.filter(status='submitted',
+                                                                           submitted_at__range=(start_datetime, end_datetime)
+                                                                           )
+
+        log.info(f"Attempting to re-submit {len(attempts_to_retry)} failed SoftwareSecurePhotoVerification submissions; "
+                 f"\nIn date range: {start_datetime} to {end_datetime}"
+                 f"\n\nDATA:"
+                 f"{attempts_to_retry}"
```

Comment:
The batch size is 300, so we probably don't want to print out all 300 of those. It might be better to just print the number of attempts that are being retried.

Reply:
Oh right, forgot to mark that as a debug thing. I'm just using to test stuff locally, I'll remove it before merging.